internal/cli/cluster/connectinfo.go

```diff
@@ -0,0 +1,494 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"tidbcloud-cli/internal"
+	"tidbcloud-cli/internal/config"
+	"tidbcloud-cli/internal/flag"
+	"tidbcloud-cli/internal/service/cloud"
+	connectInfoModel "tidbcloud-cli/pkg/tidbcloud/connect_info/models"
+
+	clusterApi "github.com/c4pt0r/go-tidbcloud-sdk-v1/client/cluster"
+
+	"github.com/juju/errors"
+	"github.com/spf13/cobra"
+)
+
+// xxxDisplayName is used to display in interactive mode
+// xxxInputName is used to input in non-interactive mode and display in help message
+const (
+	GeneralParameterID              string = "standard_connection_parameter"
+	GeneralParameterDisplayName     string = "General Parameter"
+	GeneralParameterInputName       string = "general_parameter"
```

Comment:
just "general"?

Reply:
@zhangyangyu Thanks for your review. I have renamed it to "General".